mlflow/xgboost.py

```diff
@@ -286,12 +309,11 @@ def load_model(model_uri):
                       `Referencing Artifacts <https://www.mlflow.org/docs/latest/tracking.html#
                       artifact-locations>`_.
 
-    :return: An XGBoost model (an instance of `xgboost.Booster`_)
+    :return: An XGBoost model. An instance of either `xgboost.Booster`_ or XGBoost scikit-learn
+             models, depending on the saved model class specification.
     """
     local_model_path = _download_artifact_from_uri(artifact_uri=model_uri)
-    flavor_conf = _get_flavor_configuration(model_path=local_model_path, flavor_name=FLAVOR_NAME)
-    xgb_model_file_path = os.path.join(local_model_path, flavor_conf.get("data", "model.xgb"))
-    return _load_model(path=xgb_model_file_path)
+    return _load_model(path=local_model_path)
```

Comment:
@dbczumar @harupy Is it a good idea to apply `_XGBModelWrapper` here to unify the input format no matter what XGBoost model is loaded?

Reply:
I don't think we should do that because `mlflow.xgboost.log_model` should return what the user logs (e.g. if the user logs `xgboost.XGBClassifier`, `xgboost.XGBClassifier` should be returned).

Reply:
Thanks for your explanation! I think it is good to leave as it is.